mica-core/src/main/java/org/obiba/mica/dataset/domain/DatasetVariable.java

```diff
@@ -39,7 +39,7 @@
   public static final String OPAL_HARMONIZATION_TABLE_PREFIX = "Harmonization";
 
   public enum Type {
-    Collection,  // variable extracted from a collection dataset
+    Collection,  // variable extracted from a collected dataset
```

Comment:
I should have add description : I will add this later (I make the ticket line by line to have many little pull requests (as little as possible))